scripts/verify-replace.sh

```diff
@@ -5,7 +5,11 @@ GH_BASE_URL_KS=https://github.com/kubesaw/
 GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
 declare -a REPOS=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
 C_PATH=${PWD}
-ERRORLIST=()
+ERRORREPOLIST=()
+ERRORFILELIST=()
+STDOUTFILELIST=()
+GOLINTREGEX="[\s\w.\/]*:[0-9]*:[0-9]*:[\w\s)(*.\`]*"
+LINTERERRORFILE=$(mktemp ${TMP_DIR}LinterError.XXX)
```

Comment:
the same with grouping the files under the same directory
```suggestion
LINTERERRORFILE=$(mktemp ${BASE_REPO_PATH}/LinterError.XXX)
```

Reply:
okay yes i agree , updated and grouped it